conf/evolutions/reversions/023-drop-datastore-type.sql

```diff
@@ -0,0 +1,9 @@
+START TRANSACTION;
+
+DROP VIEW webknossos.datastores_;
+ALTER TABLE webknossos.datastores add typ webknossos.DATASTORE_TYPE NOT NULL DEFAULT 'webknossos-store',
+CREATE VIEW webknossos.dataStores_ AS SELECT * FROM webknossos.dataStores WHERE NOT isDeleted;
+
+UPDATE webknossos.releaseInformation set schemaVersion = 22;
```

Comment:
this is now evolution `23`

Reply:
so the reversion has to set the schemaVersion back to 22 :)

Reply:
I mixed it with the actual evolutions, didn't see it's the reversion, sorry

Reply:
It’s very easy to miss, maybe the naming scheme should be different or something. Either way, thanks for the review :)